src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -64,7 +99,40 @@ public void testVCFModeIsConcordantWithGATK3Results() throws Exception {
         runCommandLine(args);
 
         final double concordance = calculateConcordance(output, gatk3Output);
-        Assert.assertTrue(concordance >= 0.99, "Concordance with GATK 3.4 in VCF mode is < 99%");
+        Assert.assertTrue(concordance >= 0.99, "Concordance with GATK 3.5 in VCF mode is < 99%");
+    }
+
+    /*
+ * Test that in VCF mode we're >= 99% concordant with GATK3.5 results
+ */
+    @Test
+    public void testVCFModeIsConcordantWithGATK3_5ResultsAlleleSpecificAnnotations() throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File output = createTempFile("testVCFModeIsConcordantWithGATK3_5ResultsAlleleSpecificAnnotations", ".vcf");
+
+        //Created by running
+        //java -jar ~/bin/GenomeAnalysisTK-3.5.0/GenomeAnalysisTK.jar -T HaplotypeCaller \
+        // -I ./src/test/resources/large/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam \
+        // -R src/test/resources/large/human_g1k_v37.20.21.fasta -L 20:10000000-10100000 \
+        // --out as.gatk3.5.noDownsample.vcf -G StandardHC -G Standard -G AS_Standard \
```

Comment:
yes, the difference stems from the fact that gatk4 interprets collection arguments as 'add to the list of defaults' while gatk3 interpreted them as 'replace the list of defaults with the provided list'. We may (probably should) change that but that's a different ticket.